quickstart/scripts/train-local.sh

```diff
@@ -30,3 +30,5 @@ gcloud ai-platform local train \
         --package-path=${PACKAGE_PATH} \
```

Comment:
Do we need Kokoro test for quick start?

Reply:
Yes for sure. We will add it in a separate PR (perhaps we can work on it together as our standard for writing unit tests).